carbonmark-api/src/plugins/open-api.ts

```diff
@@ -31,6 +41,20 @@ For a developer guides and example implementations, or to learn more about Carbo
     /** The API version */
     version: packageJson.version,
   },
+  components: {
+    schemas: {
```

Comment:
I'm afraid this makes our docs more confusing, because the schemas are all listed in the nav menu, but they don't clarify where in the api they are found, nor does the endpoint reference the relevant schema.

I think $refs would be the solution here, and I believe stoplight supports $refs, but for now lets just tear them out from this file to remove them from the nav menu until we figure out a good solution

Reply:
The main issue here is that this object is where `feTS` extracts the models used to generate the types in the Carbonmark app :(

I initially tried to use #refs however after a couple of hours sunk I couldn't get it to play nicely with Typebox (I was even messing around with schema resolvers etc..)